src/style.css

```diff
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --white: #fff;
+  --blue-accent: #00acdc;
+  --gray-elements: #f3f3f3;
+  --yellow-stars: #ffde6a;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars :last-child {
+  margin: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2) {
  background-image: url(./images/star-active.svg);
}
```
<img width="586" alt="image" src="https://github.com/user-attachments/assets/368315a7-f8ca-4995-b7de-810135bddf1d">